src/unit/Makefile

```
@@ -114,19 +114,38 @@ WRAP_DIR := wrapped
+# Homebrew prefix locations for both Apple Silicon (/opt/homebrew) and Intel (/usr/local).
+BREW_LLVM_BIN := $(firstword $(wildcard /opt/homebrew/opt/llvm/bin /usr/local/opt/llvm/bin))
+ifneq ($(shell command -v llc 2>/dev/null),)
+  LLC ?= llc
+  LLVM_DIS ?= llvm-dis
+  LLVM_OBJCOPY ?= llvm-objcopy
+else ifneq ($(BREW_LLVM_BIN),)
+  LLC ?= $(BREW_LLVM_BIN)/llc
+  LLVM_DIS ?= $(BREW_LLVM_BIN)/llvm-dis
+  LLVM_OBJCOPY ?= $(BREW_LLVM_BIN)/llvm-objcopy
+else
+  $(error LLVM tools not found. Install via Homebrew: brew install llvm)
```

checkmake (0.3.2)

[warning] Target "$(error LLVM tools not found. Install via Homebrew" should be declared PHONY.

(phonydeclared)